src/components/organisms/UiMultipleChoices/UiMultipleChoices.stories.ts

```diff
@@ -0,0 +1,333 @@
+import { ref } from 'vue';
+import type {
+  Meta,
+  StoryObj,
+} from '@storybook/vue3';
+import { useArgTypes } from '@sb/helpers';
+import {
+  UiAlert,
+  UiBulletPoints,
+  UiHeading,
+  UiMultipleChoices,
+  UiSidePanel,
+  UiText,
+} from '@index';
+import UiMultipleChoicesItem from '@/components/organisms/UiMultipleChoices/_internal/UiMultipleChoicesItem.vue';
+import type { MultipleChoicesProps } from '@index';
+import './UiMultipleChoices.scss';
+
+type MultipleChoicesArgsType = MultipleChoicesProps;
+type MultipleChoicesMetaType = Meta<MultipleChoicesArgsType>;
+type MultipleChoicesStoryType = StoryObj<MultipleChoicesArgsType>;
+
+const { argTypes } = useArgTypes(UiMultipleChoices);
+const isSidePanelOpen = ref(false);
+
+function handleButtonInfoClick() {
+  isSidePanelOpen.value = !isSidePanelOpen.value;
+}
+
+const buttonInfoItems = [
+  {
+    id: 'i-have-diabetes',
+    label: 'I have diabetes',
+    translation: { info: 'What does it mean?' },
+    buttonInfoAttrs: { onClick: handleButtonInfoClick },
+    iconInfoAttrs: { 'data-testid': 'info-icon' },
+  },
+  {
+    id: 'i-have-hypertension',
+    label: 'I have hypertension',
+  },
+  {
+    id: 'i-have-hypertension',
+    label: 'I have high cholesterol',
+    translation: { info: 'How to check it?' },
+    buttonInfoAttrs: { onClick: handleButtonInfoClick },
+    iconInfoAttrs: { 'data-testid': 'info-icon' },
+  },
+];
+
+const meta = {
+  title: 'Organisms/MultipleChoices',
+  component: UiMultipleChoices,
+  args: {
+    modelValue: [],
+    items: [
+      {
+        label: 'I have diabetes',
+        textLabelAttrs: { 'data-testid': 'label-text' },
+      },
+      {
+        label: 'I have hypertension',
+        textLabelAttrs: { 'data-testid': 'label-text' },
+      },
+      {
+        label: 'I have high cholesterol',
+        textLabelAttrs: { 'data-testid': 'label-text' },
+      },
+    ],
+    options: [
+      {
+        label: 'Yes',
+        value: 'present',
+      },
+      {
+        label: 'No',
+        value: 'absent',
+      },
+      {
+        label: "Don't know",
+        value: 'unknown',
+      },
+    ],
+    hint: 'Select one answer in each row',
+    touched: false,
+    invalid: true,
+    alertHintAttrs: { 'data-testid': 'alert-hint' },
+  },
+  argTypes: {
+    ...argTypes,
+    modelValue: { control: 'object' },
+    items: { control: 'object' },
+    options: { control: 'object' },
+  },
+  decorators: [ () => ({
```

Comment:
Why do we need this decorator?

Reply:
I  added the decorator only to add space between frame and content like it is in Question